MediaBrowser.Providers/MediaInfo/MediaInfoResolver.cs

```diff
@@ -148,15 +148,19 @@ public abstract class MediaInfoResolver
 
             // Check if video folder exists
             string folder = video.ContainingFolderPath;
-            if (!Directory.Exists(folder))
+            if (!directoryService.PathExists(folder))
             {
                 return Array.Empty<ExternalPathParserResult>();
             }
 
             var externalPathInfos = new List<ExternalPathParserResult>();
```

Comment:
Not related to your PR, but it's a bit awkward that there's a new List here when we return an empty array just a few lines down